kopf/k8s/classes.py

```diff
@@ -0,0 +1,48 @@
+import json
+from typing import Type
+
+import pykube
+
+from kopf.k8s import config
+
+
+# TODO: this mixin has to be migrated into pykube-ng itself.
```

Comment:
feel free to open a PR for pykube-ng :smile:

Reply:
I will for sure. First of all, let's see how it works in this mode. And second:

With the official client, there was an issue when the switch the Content-Type from merge-patch format (dicts of fields) to JSON-patch format (lists of ops) — see #134 and kubernetes-client/python#866.

For Kopf, merge-patch is needed and is sufficient. But for a generic k8s client, both formats should be supported.